draft-enghardt-panrg-path-properties.md

```diff
@@ -147,57 +149,39 @@ Conversely, the selection of a protocol may influence the devices that will be i
 For example, a 0-RTT Transport Converter {{I-D.ietf-tcpm-converters}} will be involved in a path only when invoked by a host; such invocation will lead to the use of MPTCP or TCPinc capabilities while such use is not supported via the default forwarding path. Another example of traffic policies is a connection which may be composed of multiple streams; each stream with specific service requirements. A host may decide to invoke a given service function (e.g., transcoding) only for some streams while others are not processed by that service function.
 
 
-# Path Property Classification
+# Examples of Path Properties
+
+This Section gives some examples of Path Properties which may be useful, e.g., for the use cases described in {{use-cases}}.
 
-Path properties may be relatively dynamic, e.g. median one-way delay of packets sent over a specific path within the last second.
-Path properties may also be non-dynamic, i.e., change less frequently, i.e., on a timescale of seconds or more.
+Path properties may be relatively dynamic or non-dynamic.
+Dynamic path properties relate to the transmission of an individual packet or of a flow over a subpath, e.g., the one-way delay of a packet sent over a specific path.
+Non-dynamic path properties relate to individual path elements or subpaths.
```

Comment:
I can't come up with another good naming than dynamic vs non-dynamic. I think if we use "flow", e.g., "Flow Properties", then this would indicate that other properties are not defined for a flow, which is not necessarily always the case. As an example, we defined "Monetary cost" and "Service function" as non-dynamic properties which relate to a flow.

Instead of dynamic, we could maybe label them as short-term and long-term properties and refer to the expected time for which the property is useful.

Reply:
You're right, both dynamic and non-dynamic properties may relate to the transmission of a specific flow.
What I meant here was: Dynamic properties express something related to the performance of transmitting this flow over these path elements. Right? All of the current dynamic properties directly express some kind of performance measurement, like how long the transmission took or how much data can be transmitted over time. And none of the non-dynamic properties directly expresses some performance measurement.
So maybe something like "performance properties" or "transmission performance properties"?

I think we used "dynamic" or "short-term" for these properties but actually meant that a specific measurement of these properties is a momentary thing, and performance changes over time, so the next measurement might be different.
However, I'm rethinking "dynamic" or "short-term" because the Link Capacity is actually not something I'd expect to change all the time. For a single link, it might not actually change that much (like for an Ethernet cable), and for a path, it might only change when the path changes, similar to what we have for "non-dynamic" properties right now.

Of course, our last resort could be to just get rid of all the classifications for now, if we can't decide ;)